engine/src/main/scala/cromwell/webservice/MetadataBuilderActor.scala

```diff
@@ -128,41 +128,60 @@ object MetadataBuilderActor {
       case s => Option(s.toInt)
     }
   }
+
+  private case object IndexedEmptyJson extends IndexedJsonValue {
+    override def index: Option[Int] = None
+    override def withIndex(index: String): IndexedJsonValue = this
+    override def toJson: JsValue = throw new IllegalStateException("IndexedEmptyJson cannot be converted to Json")
+  }
```

Comment:
Hmm yeah actually it might be simpler with a `Monoid`
Also - now I'm going to go home and cry for having said that